zeppelin-server/src/main/java/org/apache/zeppelin/rest/InterpreterRestApi.java

```diff
@@ -90,6 +90,9 @@ public Response newSettings(String message) {
     try {
       NewInterpreterSettingRequest request =
           gson.fromJson(message, NewInterpreterSettingRequest.class);
+      if (request == null) {
+        return new JsonResponse<>(Status.BAD_REQUEST).build();
```

Comment:
@jongyoul The error code 400 is returned when the input json is empty and we expect something from the request.Should I go ahead and update the error code in the doc for this case?